storage-node-v2/src/command-base/ApiCommandBase.ts

```diff
@@ -0,0 +1,135 @@
+import { Command, flags } from '@oclif/command'
+import { createApi } from '../services/runtime/api'
+import { getAccountFromJsonFile, getAlicePair } from '../services/runtime/accounts'
+import { KeyringPair } from '@polkadot/keyring/types'
+import { ApiPromise } from '@polkadot/api'
+import logger from '../services/logger'
+import ExitCodes from './ExitCodes'
+import { CLIError } from '@oclif/errors'
+import { Input } from '@oclif/parser'
+
+/**
+ * Parent class for all runtime-based commands. Defines common functions.
+ */
+export default abstract class ApiCommandBase extends Command {
+  private api: ApiPromise | null = null
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    dev: flags.boolean({ char: 'm', description: 'Use development mode' }),
+    apiUrl: flags.string({
+      char: 'u',
+      description: 'Runtime API URL. Mandatory in non-dev environment. Default is ws://localhost:9944',
```

Comment:
```suggestion
      description: 'Runtime API URL. Default is ws://localhost:9944',
      default: 'ws://localhost:9944',
```

Doesn't appear to be required when --dev arg is not specified, and might make more sense to set default here rather than in `init()` function.

Reply:
Ok